Library/Formula/twister.rb

```diff
@@ -0,0 +1,56 @@
+require "formula"
+
+class Twister < Formula
+  homepage "http://twister.net.co"
+  url "https://github.com/miguelfreitas/twister-core/archive/v0.9.28.tar.gz"
+  sha1 "53b03636ae4d7539002fe9c3cf2dd58e9657ac1e"
+
+  head do
+    url "https://github.com/miguelfreitas/twister-core.git"
+  end
+
+  depends_on "boost" => :build
+  depends_on "miniupnpc" => :build
+  depends_on "openssl" => :build
+  depends_on "berkeley-db4" => :build
```

Comment:
deps are needed at runtime. They usually show up with ` otool `. Build deps are only needed during the compile stage, commonly things like ` pkg-config `, ` cmake `, ` libtool `, etc. Because we bottle things, when users install from a bottle, they don't have to install the build-time deps.

I'd be surprised if ` miniupnpc ` wasn't run-time, given its general purpose, and the fact the two projects Twister was spun off of were Bitcoin and Libtorrent, and Bitcoin has a ` configure ` switch for ` miniupnpc `. It might be a case that it needs specifically enabling somewhere.